lib/evaluation/handlers/uriEvaluationHandler.ts

```diff
@@ -80,6 +79,21 @@ export class UriEvaluationHandler extends AbstractEvaluationHandler {
     }
   }
 
+  private static fetchVcUris(vc: InternalVerifiableCredential) {
```

Comment:
The naming is off. Fetch in the context or URIs suggests that you will be actually getting the content of the URI.
Suggest to rename it to something like `buildVcContextAndSchemaUris()` or something

Reply:
changed